setup.py

```diff
@@ -35,6 +35,8 @@
 
 test_requirements = ['pytest', ]
 
+extras_requirements = {'test': ['scanpy', 'louvain', ]}
```

Comment:
What happens if we list these as test_requirements rather than extras_requirements?

Reply:
It doesn't work. Tox doesn't handle well test_requirements ...
Here is a description of the problem, with a solution using extras (deps= .[test])
https://stackoverflow.com/questions/29870629/pip-install-test-dependencies-for-tox-from-setup-py

I rather use the suggested syntax from tox documentation to install extras: extras=test